### DISCUSSION TITLE
Add button "run query" to flame graph panel to test query changes

### DISCUSSION THREAD

**Discussion by tomershafir** [grafana/grafana#72560] [Dashboards, Panels & Visualizations]
On OSS grafana flame graph panel with phlare datasource, when you change a query, you cannot inspect the new query result directly, either automatically or on demand.

Add a button "run query", like in Prometheus datasource.

Env:
Grafana v9.5.7
Chrome 101.0.4951.54 (Official Build) (arm64)
MacOS Monterey 12.2

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82490. Feel free to continue the discussion around this there. Thank you!
